src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementService.java

```diff
@@ -140,7 +144,7 @@ public ClaimDocument logUploadDocumentFailure(
             return resource.getByteArray();
         } catch (Exception ex) {
             throw new DocumentManagementException(
-                String.format("Unable to download document %s from document management", baseFileName), ex);
+                String.format("Unable to download document %s from document management.", baseFileName), ex);
```

Comment:
would be better to pass ex directly as DocumentManagementException(ex). as @recover logging confirms that it has finally failed after retry.

Reply:
I think its best to throw a specific custom checked exception coz the root exception could have been anything.